util/manually-added-ecqm-data.json

```diff
@@ -0,0 +1,329 @@
+[
+  {
+    "eMeasureId": "CMS117v5",
+    "strata": [
+      {
+        "name": "receivedVaccinesOrExposure",
+        "description": "Children who have evidence showing they received recommended vaccines, had documented history of the illness, had a seropositive test result, or had an allergic reaction to the vaccine by their second birthday"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS123v5",
+    "strata": [
+      {
+        "name": "receivedExams",
+        "description": "Patients who received visual, pulse and sensory foot examinations during the measurement period"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS124v5",
+    "strata": [
+      {
+        "name": "screenedForCervicalCancer",
+        "description": "Women with one or more screenings for cervical cancer. Appropriate screenings are defined by any one of the following criteria:\r\n\r\n- Cervical cytology performed during the measurement period or the two years prior to the measurement period for women who are at least 21 years old at the time of the test\r\n \r\n- Cervical cytology/human papillomavirus (HPV) co-testing performed during the measurement period or the four years prior to the measurement period for women who are at least 30 years old at the time of the test"
+      }
+    ]
+  },
+  {
+    "eMeasureId": "CMS128v5",
+    "strata": [
+      {
+        "name": "continuousTreatmentAtLeast84Days",
+        "description": "Patients who have received antidepressant medication for at least 84 days (12 weeks) of continuous treatment during the 114-day period following the Index Prescription Start Date"
+      },
+      {
+        "name": "continuousTreatmentAtLeast180Days",
+        "description": "Patients who have received antidepressant medications for at least 180 days (6 months) of continuous treatment during the 231-day period following the Index Prescription Start Date"
+      }
+    ],
+    "overallAlgorithm": "simpleAverage"
+  },
+  {
+    "eMeasureId": "CMS136v6",
+    "strata": [
+      {
+        "name": "hadPractitionerVisit",
+        "description": "Patients who had at least one face-to-face visit with a practitioner with prescribing authority within 30 days after the IPSD."
+      },
+      {
+        "name": "hadPractitionerVisitAndFollowups",
+        "description": "Patients who had at least one face-to-face visit with a practitioner with prescribing authority during the Initiation Phase, and at least two follow-up visits during the Continuation and Maintenance Phase. One of the two visits during the Continuation and Maintenance Phase may be a telephone visit with a practitioner."
+      }
+    ],
+    "overallAlgorithm": "simpleAverage"
+  },
+  {
+    "eMeasureId": "CMS137v5",
+    "strata": [
+      {
+        "name": "treatmentWithin14Days",
+        "description": "Patients who initiated treatment within 14 days of the diagnosis"
+      },
+      {
+        "name": "treatmentPlusServicesWithin30",
```

Comment:
Might be worth adding "days" to the end of it? Ie `treatmentPlusServicesWithin30DaysOfVisit`?